release-1.8/release_notes_draft.md

```diff
@@ -69,6 +69,9 @@ API surface for promotion.
  apps/v1beta2 group.
  - The pod.alpha.kubernetes.io/initialized has been removed.
 
+### Scheduling
+- Opaque Integer Resources (OIRs) are deprecated and will be removed in
+  v1.9. Extended Resources (ERs) are a drop-in replacement for OIRs.
```

Comment:
May be add the fact that the resource prefixes have to change for ERs when porting OIRs to ERs?